sql-plugin/src/main/320until330-cdh/scala/com/nvidia/spark/rapids/shims/OrcShims.scala

```diff
@@ -17,15 +17,25 @@ package com.nvidia.spark.rapids.shims
 
 import org.apache.orc.Reader
 
+import org.apache.spark.sql.execution.datasources.orc.OrcUtils
```

Comment:
seems like this could go into the 321cdh directory

Reply:
done